README.md

```diff
@@ -39,18 +39,19 @@ require 'rbslicer'
 # Configure the client
 client = SlicingDice.new(master_key: "API_KEY", uses_test_endpoint: true)
 
-# Indexing data
-index_data = {
+# Inserting data
+insert_data = {
     "user50@slicingdice.com" => {
         "age" => 22
     }
 }
-auto_create_fields = true
-client.index(index_data, auto_create_fields)
+auto_create_columns = true
+client.insert(insert_data, auto_create_columns)
```

Comment:
The auto-create-columns parameter should be removed.
Check this PR (https://github.com/SlicingDice/slicingdice-ruby/pull/10)